### PR TITLE
V2 for release 2.4

### PR DESCRIPTION
**Pull request by shasibhusanJena**
- removed line from app.py default AssertionException block - added a…  …
… separate code block on config.py file to validate this condition.
-- doing a comparison with connectors_config. the value so that, don't have to loop through to make a conditional check "Okta" type

### CODE REVIEW

**Comment by adorton-adobe** [comment]
@shasibhusanJena You don't need to make new PR when you make new commits.  Just commit to the branch the PR is coming from and they'll be picked up here.

**Comment by shasibhusanJena** [comment]
oh, I was not knowing about it before, that will be easy then. thanks

File: user_sync/config.py [hunk 1]
```diff
@@ -323,6 +323,11 @@ def get_directory_connector_options(self, connector_name):
         """
         options = {}
         connectors_config = self.get_directory_connector_configs()
+
+        if ( 'okta' in connectors_config.value) and (connector_name == 'ldap'):
+            if connectors_config.accessed_keys is not None:
+                raise AssertionException("Failed to match request : for Okta Connector type receiving LDAP as input type from commnand line")
```
Comment (adorton-adobe) [inline user_sync/config.py]: If you make the check more generic you'll need to make this message generic as well.  I'd do something like this:

```python
raise AssertionException("Config file must be specified for connector type '{}'".format(connector_name))
```

File: user_sync/config.py [hunk 2]
```diff
@@ -323,6 +323,11 @@ def get_directory_connector_options(self, connector_name):
         """
         options = {}
         connectors_config = self.get_directory_connector_configs()
+
+        if ( 'okta' in connectors_config.value) and (connector_name == 'ldap'):
+            if connectors_config.accessed_keys is not None:
```
[the remark this answers is not in the document]
Comment (shasibhusanJena) [reply user_sync/config.py]: connectors_config.accessed_keys that were to find the type as csv, okta, LDAP. I was one layer of the check, which is not required now, so removing it.

File: user_sync/config.py (hunk 1 above)
Comment (shasibhusanJena) [reply user_sync/config.py]: modified. please review

**Pull request merged**